### PR TITLE
Add method for appending version with .0s

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Add method in helpers that can be used to append version with .0s, for
example 1.2 can become 1.2.0.

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by Fatme** [comment]
:+1:

File: helpers.ts [hunk 1]
```diff
@@ -137,3 +137,12 @@ export function trimSymbol(str: string, symbol: string) {
 
 	return str;
 }
+
+export function appendZeroesToVersion(version: string, requiredVersionLength: number): string {
+	let zeroesToAppend = requiredVersionLength - version.split(".").length;
+	for (var index = 0; index < zeroesToAppend; index++) {
```
Comment (ligaz) [inline helpers.ts]: `let` me be.

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**